src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -1550,24 +1493,34 @@ public boolean exists(Pool pool) {
     }
 
     public void calculateConsumedForOwnersPools(Owner owner) {
-        String stmt = "update Pool p set p.consumed = coalesce(" +
-            "(select sum(quantity) from Entitlement ent where ent.pool.id = p.id),0) " +
-            "where p.owner = :owner";
+        String jpql = """
+            UPDATE Pool p
+            SET p.consumed = coalesce(
+                (SELECT sum(quantity) FROM Entitlement ent
+                 WHERE ent.pool.id = p.id), 0)
+            WHERE p.owner = :owner""";
+
+        javax.persistence.Query query = getEntityManager().createQuery(jpql);
```

Comment:
With the removal of the Hibernate Query import, we can use this Query class directly.

Note this occurs in a few places

Reply:
done